xbmc/guilib/StereoscopicsManager.cpp

```diff
@@ -509,6 +509,8 @@ void CStereoscopicsManager::OnPlaybackStarted(void)
   STEREOSCOPIC_PLAYBACK_MODE playbackMode = (STEREOSCOPIC_PLAYBACK_MODE) CSettings::GetInstance().GetInt(CSettings::SETTING_VIDEOPLAYER_STEREOSCOPICPLAYBACKMODE);
   RENDER_STEREO_MODE mode = GetStereoMode();
 
+  g_infoManager.UpdateAVInfo();
```

Comment:
how would player be informed when to proceed and by whom? Manager only tells graphicsContext to apply a 3D mode, it does not know when the mode actually got applied/resolution switched. Also, will graphicsContext at this stage already know about video playback metadata in order to select the correct refresh rate f.e. when switching from 3D to 2D, or who is responsible to inform it about the correct resolution and refresh rate? (I'm not into that code)